packages/arb-validator-core/mockbridge/messagesChallenge.go

```diff
@@ -123,6 +123,15 @@ func (c *MessagesChallenge) OneStepProofTransactionMessage(
 	return nil
 }
 
+func (c *MessagesChallenge) OneStepProofTransactionBatchMessage(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [method receiver 'c' is not referenced in method's body, consider removing or renaming it as _](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)